tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -222,10 +237,19 @@ static int getSortKey(double minZoom, int kindRank, int populationRank, long pop
     ), 0);
 
   public void processOsm(SourceFeature sf, FeatureCollector features) {
-    if (!sf.isPoint() || !sf.hasTag("name")) {
+    if (!sf.isPoint() || !sf.hasTag("name") || !sf.hasTag("place")) {
       return;
     }
 
+    try {
+      Optional<String> code = countryCoder.getCountryCode(sf.latLonGeometry());
```

Comment:
Good use of country coder, though I wonder if it is going to be accurate for all regions especially if they have "odd" geometries like exclaves not fully contained? Anyways, this is better than nothing for now

Reply:
Not fully sure here either. There are also different country codes for example for France (Europe+overseas) and Metropolitan France (Europe only), so one has to be careful with this...